doc/api/_toc.md

```diff
@@ -24,6 +24,7 @@
 * [HTTP](http.html)
 * [HTTPS](https.html)
 * [Modules](modules.html)
+* [N-API](n-api.html)
```

Comment:
Should be titled something like "C/C++ Addons (N-API)", so that it is adjacent to the other "C/C++ Addons" topic?